### PR TITLE
feat(ArtNet): Adds support for ArtNet.  

### PR DESCRIPTION
**Pull request by alexanderson1993**
To configure, set the `ARTNET_DMX_ADDRESS` environment variable to the IP address of the ArtNet controller.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -27,52 +28,63 @@ if (debugMode) {
...
-      const output = new DmxOutput(
-        new EnttecUsbProMk2Driver(usbproSerialport),
-        1
-      )
+      const output = new DmxOutput(driver, 1)
       output.start(1000 / 30)
 
       bridge.setOutput(output)
 
       console.log('Set enttec DMX USB Pro as the output for fivetwelve')
```
Comment (TimPietrusky) [inline src/index.js]: This should be moved into line 72, as it is only related to the enttec controller, but not for the general driver.

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
I have one change request!

File: src/index.js (hunk 1 above)
Comment (alexanderson1993) [reply src/index.js]: That should do it.

**Review by TimPietrusky — APPROVED** [review]
LGTM

**Comment by TimPietrusky** [comment]
@alexanderson1993 thank you very much!

**Pull request merged**